flask_resources/content_negotiation.py

```diff
@@ -82,39 +83,23 @@ def content_negotiation(f):
 
     @wraps(f)
     def inner(self, *args, **kwargs):
-        # Content-Type
-        # Check if content-type can be treated otherwise, fail fast
-        # Serialization is checked per function due to lack of
-        # knowledge at this point
-        payload_mimetype = request.content_type
-        allowed_mimetypes = self._request_loaders.keys()
-        if payload_mimetype not in allowed_mimetypes:
-            raise UnsupportedMimetypeError(
-                header="Content-Type",
-                received_mimetype=payload_mimetype,
-                allowed_mimetypes=allowed_mimetypes,
-            )
-
-        # Accept
-        allowed_mimetypes = self._response_handlers.keys()
+        """Wrapping method.
+
+        :params self: ItemView or ListView class
```

Comment:
comment: SingletonView also no? even if its not fully implemented yet.

Reply:
Good catch, will add!